images/api.Dockerfile

```
@@ -14,10 +14,10 @@ WORKDIR /app
 
 COPY --from=builder /go/src/github.com/tektoncd/hub/api/api-server /app/api-server
 
-# For each new version, doc has to be copied 
+# For each new version, doc has to be copied
 COPY gen/http/openapi3.json /app/docs/openapi3.json
```

COPY gen/http/openapi3.json /app/docs/openapi3.json
```suggestion
COPY api/gen/http/openapi3.json /app/docs/openapi3.json
```